docs/trace/getting-started/README.md

```diff
@@ -50,15 +50,39 @@ What does the above program do?
 
 The program creates an `ActivitySource` which represents an [OpenTelemetry
 Tracer](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#tracer).
+
+```csharp
+private static readonly ActivitySource MyActivitySource = new ActivitySource(
+    "MyCompany.MyProduct.MyLibrary");
+```
+
 The `ActivitySource` instance is used to start an `Activity` which represents an
 [OpenTelemetry
 Span](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span).
+
+```csharp
+using (var activity = MyActivitySource.StartActivity("SayHello"))
+{
+    activity?.SetTag("foo", 1);
```

Comment:
since the code here is showing starting activity + setting tags, the text above it need to reflect that.

Something along the lines  "The `ActivitySource` instance is used to start an `Activity`, and set several `Tags` on it"